pyxem/tests/signals/test_differential_phase_contrast.py

```diff
@@ -469,3 +469,83 @@ def test_retain_axes_manager(self):
         assert s_load.axes_manager[2].units == "b"
         assert s_load.axes_manager[1].name == "e"
         assert s_load.axes_manager[2].name == "f"
+
+class TestPhaseRetrieval:
+
+    def setup_method(self):
+        # construct the surface, two point with Gaussian distribution
+        rng = np.linspace(-20,10,num=512)
```

Comment:
why have you called this `rng`? Makes me think of Random Number Generat(or/ed)

Reply:
Use `coords` now as coordinates

Reply:
great, was worried I was being slow, but I think `coords` is clearer